tests/unit/src/flixel/FlxCameraTest.hx

```diff
@@ -39,16 +39,19 @@ class FlxCameraTest extends FlxTest
 	@Test
 	function testDefaultCameras():Void
 	{
-		Assert.areEqual(FlxG.cameras.defaults, FlxCamera.defaultCameras);
+		@:privateAccess
```

Comment:
Why start using privateAccess in here?

Reply:
to remove the deprecation warnings

Reply:
That feels very hacky. I think I'd rather have the deprecation warnings.

Reply:
both seem hacky, but sure